osidb/models.py

```diff
@@ -1130,6 +1139,19 @@ def _validate_ps_module_new_flaw(self):
                     f"for flaw with bz_id {bz_id}."
                 )
 
+    def _validate_tracker_flaw_accesses(self):
+        """
+        Check whether an affect of an embargoed flaw has a public tracker.
+        """
+        if (
+            self.flaw
+            and self.flaw.is_embargoed
+            and Tracker.objects.filter(affects=self, embargoed=False)
```

Comment:
I have removed this validation, but FYI why I decided to do it backwards, I have forgotten for a minute the relational field gets me a queryset and not a list and because the embargoed annotated field are only available through the ModelManager in direct queries I have made it the confusing way, but your solution is definitely better.